R/author.R

```diff
@@ -84,17 +86,17 @@ challenge_legacy_author_fields <- function(d = proj_desc()) {
     return(invisible())
   }
 
-  ui_oops("
-    Found legacy {ui_field('Author')} and/or {ui_field('Maintainer')} field \\
-    in DESCRIPTION.
-    usethis only supports modification of the {ui_field('Authors@R')} field.")
-  ui_info("
-    We recommend one of these paths forward:
-    * Delete these fields and rebuild with {ui_code('use_author()')}.
-    * Convert to {ui_field('Authors@R')} with {ui_code('desc::desc_coerce_authors_at_r()')},
-      then delete the legacy fields.")
-  if (ui_yeah("Do you want to cancel this operation and sort that out first?")) {
-    ui_stop("Cancelling.")
+  ui_bullets(c(
+    "x" = "Found legacy {.field Author} and/or {.field Maintainer} field in
+           DESCRIPTION.",
+    " " = "usethis only supports modification of the {.field Authors@R} field.",
+    "i" = "We recommend one of these paths forward:",
+    "_" = "Delete the legacy fields and rebuild with {.fun use_author}; or",
```

Comment:
I introduce a new bullet type with shortcode `"_"`. This is for TODOs and is meant to evoke a place where you could check something off.